### PR TITLE
Prepare 0.4.0

### PR DESCRIPTION
**Pull request by therealemjy**
- bump version to 0.4.0
- remove top banner
- disable supply tab for LUNA and UST
- disable borrow tab for LUNA and UST
- prevent `Table` component from overflowing

### CODE REVIEW

File: src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx [hunk 1]
```diff
@@ -201,6 +187,24 @@ export const SupplyWithdrawUi: React.FC<ISupplyWithdrawUiProps & ISupplyWithdraw
     },
   ];
 
+  // Prevent user from being able to supply UST or LUNA
+  if (assetId !== 'ust' && assetId !== 'luna') {
```
Comment (coreyar) [inline src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx]: I took a slightly different approach here - https://github.com/VenusProtocol/venus-protocol-interface/pull/514

We don't have to take it but the idea was to start setting up feature flags so in the future we don't only have to add tokens to the disabled token list we don't have to change other application code too

Comment (therealemjy) [reply src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx]: Sorry for not seeing you opened a PR for that already.

I updated this PR to follow the same logic as yours :)

File: src/pages/Dashboard/Modals/SupplyWithdraw/index.spec.tsx [hunk 2]
```diff
@@ -83,6 +83,33 @@ describe('pages/Dashboard/SupplyWithdrawUi', () => {
   });
 
   describe('Supply form', () => {
+    it.each(['ust', 'luna'])('does not display supply tab when asset is %s', async tokenId => {
```
Comment (coreyar) [inline src/pages/Dashboard/Modals/SupplyWithdraw/index.spec.tsx]: nit: We could import `DISABLED_TOKENS` here to make this test more resilient

**Review by coreyar — APPROVED** [review]

File: src/pages/Dashboard/Modals/SupplyWithdraw/index.spec.tsx (hunk 2 above)
Comment (therealemjy) [reply src/pages/Dashboard/Modals/SupplyWithdraw/index.spec.tsx]: Ah yeah of course, good point

**Review by coreyar — APPROVED** [review]

**Pull request merged**